libs/execution_context/ebpf_core.c

```diff
@@ -1107,6 +1110,14 @@ _ebpf_core_random_uint32()
 
 static uint64_t
 _ebpf_core_get_time_since_boot_ns()
+{
+    // ebpf_query_interrupt_time_precise returns time elapsed since
+    // boot in units of 100 ns.
+    return ebpf_query_unbiased_interrupt_time_precise() * 100;
+}
+
+static uint64_t
+_ebpf_core_get_time_ns()
 {
     // ebpf_query_interrupt_time_precise returns time elapsed since
     // boot in units of 100 ns.
```

Comment:
From your explanation, this comment should be updated since it's not the time elapsed since boot if it excludes time suspended.